test/e2e/scheduler_predicates.go

```diff
@@ -177,7 +182,14 @@ var _ = Describe("SchedulerPredicates", func() {
 
 		pods, err := c.Pods(api.NamespaceAll).List(labels.Everything(), fields.Everything())
 		expectNoError(err)
-		currentlyRunningPods := len(pods.Items)
+		var currentlyRunningPods, currentlyDeadPods int
+		for _, pod := range pods.Items {
```

Comment:
Can you please extract it to the method (very similar code is in 3 places here) - sth like\
func (pod api.PodList) runningAndDealPods (int, int)